clients/fides-js/src/components/fides.css

```diff
@@ -53,18 +53,18 @@
 
   /* Everything else */
   --fides-overlay-font-family: Inter, sans-serif;
-  --fides-overlay-font-size-body-small: 0.875em;
-  --fides-overlay-font-size-body: 0.95em;
-  --fides-overlay-font-size-title: 1em;
-  --fides-overlay-font-size-buttons: 1.05em;
-  --fides-overlay-padding: 1.5em;
+  --fides-overlay-font-size-body-small: 12px;
```

Comment:
there's a reason for the `em`s here (though perhaps they should be `rem`s... it's an accessibility thing https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#strategic-unit-deployment-6

a short hand (which I will readily admit I often forget!)

font sizes: use `rem`
media queries: use `rem`
width/height: varies

Reply:
for reference, this is what happens now that these are pixels and we increase the browser font size—the background page respects the font size update, but our banner doesn't!
![image](https://github.com/ethyca/fides/assets/24641006/5392580d-3698-4c9c-a331-3120137d7d79)